Source/WebCore/page/DragController.cpp

```diff
@@ -762,10 +764,11 @@ static bool imageElementIsDraggable(const HTMLImageElement& image, const LocalFr
         return true;
 
     auto* renderer = image.renderer();
-    if (!is<RenderImage>(renderer))
+    CheckedPtr renderImage = dynamicDowncast<RenderImage>(renderer);
+    if (!renderImage)
         return false;
 
-    auto* cachedImage = downcast<RenderImage>(*renderer).cachedImage();
+    auto* cachedImage = renderImage->cachedImage();
```

Comment:
Please use `CachedResourceHandle cachedImage = ...` to protect the cachedImage given the member function calls below.